### PR TITLE
#1088 - Add lazy intersections page

### LINKED ISSUE #1088: Add lazy ellipsoid intersections to the manual
See [this notebook](https://nbviewer.jupyter.org/github/mforets/escritoire/blob/master/reachability/Ellipsoids.ipynb).

### PR DESCRIPTION
**Pull request by mforets**
Closes #1088.
 
~~disclaimer: i didn't make these new docs in my machine yet.~~ looks good now.

### CODE REVIEW

**Comment by schillic** [comment]
```
 !! failed to run code block.

ArgumentError("Package StaticArrays not found in current path:\n- Run `import Pkg; Pkg.add(\"StaticArrays\")` to install the StaticArrays package.\n") [src/man/convex_hulls.md]                                                                                          
 !! failed to run code block.

UndefVarError(:Ellipsoid) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:E₁) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:E₁) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:HPolytope) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:plot) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:E₁) [src/man/lazy_intersections.md]
 !! failed to run code block.

UndefVarError(:E₁) [src/man/lazy_intersections.md]
 !! failed to run code block.

LoadError("none", 6, UndefVarError(Symbol("@btime"))) [src/man/lazy_intersections.md]
```

File: docs/src/man/lazy_intersections.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+```@meta
+CurrentModule = LazySets.Approximations
+DocTestSetup = quote
+    using Plots, LazySets, LazySets.Approximations
+end
+```
+
+An ellipsoid $E$ can be created by giving its center $c$ and its shape matrix ``Q``,
+which should be positive definite, i.e. its eigenvaues must be positive.
+Mathematically, it is the set ``E = \\{ x \in \\mathbb{R}^n : (x-c)Q^{-1}(x-c) ≦ 1\\}``.
```
Comment (schillic) [inline docs/src/man/lazy_intersections.md]: Make it a centered `math` block and use ≦ → ≤.

**Comment by mforets** [comment]
Thanks. I've fixed those errors.

**Comment by mforets** [comment]
CI is green 💯

**Comment by mforets** [comment]
Thanks for the feedback 👍 👍  Yes, i get the same for the first formula (i supposed that i had fixed it). I agree with the other 2 comments, i'll add them in a new commit.

**Comment by mforets** [comment]
> E₁ ∩ E₂ prints a long string. Let us hide that (it does not add any value IMO).

can i show the type?

**Comment by mforets** [comment]
> The first math formula is broken into three lines for me. I do not understand why. Do you get the same?

Problem solved: markdown mode doesn't require to escape the backslashes.. `\{` is good etc. I was writing as if they were julia docstrings.

**Comment by mforets** [comment]
Interesting, i didn't know about `#hide`.

**Pull request merged**